contents/blog/the-posthog-array-1-31-0.md

```diff
@@ -0,0 +1,125 @@
+---
+date: 2021-12-15
+title: Array 1.31.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/generic-release-notes.png
+excerpt: PostHog 1.31.0 introduces Group Analytics, improved Correlation Analysis, a revamped overall user experience on Insights and 350+ more improvements and fixes.
+---
+
+
+Happy holidays from PostHog! This is our last release of the year. PostHog 1.31.0 introduces Group Analytics, improved Correlation Analysis, a revamped overall user experience on Insights and 350+ more improvements and fixes. Please note that a Postgres-based installation is no longer supported for PostHog 1.31.0.
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Do not upgrade to this version if you’re using the Postgres-based version. Upgrade to ClickHouse first. Instructions can be found <a href="/docs/self-host/migrate-from-postgres-to-clickhouse" target="_blank">here</a>.
+</blockquote>
+
+## PostHog 1.31.0 release notes
+
+> Don't see the new features on your self-hosted deployment? Remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Group Analytics](#group-analytics)
+- [Improved Correlation Analysis](#improved-correlation-analysis)
+- [Improved user experience](#improved-user-experience)
+
+### Group Analytics
+
+Introducing Group Analytics! Group Analytics enable you to perform aggregation on multiple levels aside from Users. For instance, if you have a B2B product, you will now be able not only to identify unique individual users, but users belonging to a Company. This enables analytics such as retention by company, events performed by unique companies, among many others. 
```

Comment:
CC @macobo for thoughts

Reply:
Let's add a link to the docs: https://posthog.com/docs/user-guides/group-analytics?utm_medium=docs&utm_campaign=posthog-array-1-31-0

Reply:
Or @joethreepwood do you think we should link to the blog post instead?

Reply:
Cool! I don't think we need the UTM though, we're navigating within the same website.